doc/Language/objects.pod6

```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Object Orientation in Perl 6
 
-Perl 6 is an object oriented language at its core, even though it allows you
-to write programs in other programming styles.
+Perl 6 is an object oriented language at its core, although you can write
```

Comment:
This section is about object oriented programming, so the less said about other paradigms, the better. How about:
Among the many computational models that Perl 6 supports, is object oriented programming.  Support for this style of programming is as the core of the language.